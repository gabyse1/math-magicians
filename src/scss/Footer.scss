.footer__box {
  height: 40px;
  padding: 0.5rem 1rem;
  background-color: #1173ac;
  color: #fff;
  display: flex;
  justify-content: center;
  align-items: center;
  position: absolute;
  bottom: 0;
  left: 0;
  right: 0;
  z-index: 10;
}
