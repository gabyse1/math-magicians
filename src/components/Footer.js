import '../scss/Footer.scss';

const Footer = () => (
  <footer className="footer__box">© 2022 Gabriela Sánchez Espirilla</footer>
);

export default Footer;
