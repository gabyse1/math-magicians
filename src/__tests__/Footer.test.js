import renderer from 'react-test-renderer';
import { cleanup, render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Footer from '../components/Footer';

describe('<Footer /> component', () => {
  afterEach(() => {
    cleanup();
  });

  it('Component render correctly', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Footer description exist', () => {
    render(<Footer />);
    const footerDesc = screen.getByTestId('footerDesc');
    expect(footerDesc).toBeInTheDocument();
    expect(footerDesc).toHaveTextContent('© 2022 Gabriela Sánchez Espirilla');
  });
});
