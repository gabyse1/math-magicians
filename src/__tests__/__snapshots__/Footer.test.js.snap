// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> component Component render correctly 1`] = `
<footer
  className="footer__box"
  data-testid="footerDesc"
>
  © 2022 Gabriela Sánchez Espirilla
</footer>
`;
